lib/DXIL/DxilModule.cpp

```diff
@@ -1044,14 +1055,17 @@ static void RemoveResourcesWithUnusedSymbolsHelper(
   for (auto gv : eraseList) {
     gv->eraseFromParent();
   }
+  return modif;
 }
 } // namespace
 
-void DxilModule::RemoveResourcesWithUnusedSymbols() {
-  RemoveResourcesWithUnusedSymbolsHelper(m_SRVs);
-  RemoveResourcesWithUnusedSymbolsHelper(m_UAVs);
-  RemoveResourcesWithUnusedSymbolsHelper(m_CBuffers);
-  RemoveResourcesWithUnusedSymbolsHelper(m_Samplers);
+bool DxilModule::RemoveResourcesWithUnusedSymbols() {
+  bool modif = false;
```

Comment:
Same suggestion here:
```suggestion
  bool Changed = false;
```